.github/workflows/wheel_tests_and_release.yml

```diff
@@ -211,9 +211,8 @@ jobs:
           SK_VERSION=$(git describe --tags)
           python setup.py sdist
           ls -la ${{ github.workspace }}/dist 
+          twine upload ${{ github.workspace }}/dist/*.whl
```

Comment:
```suggestion
          # We prefer to release wheels before source because otherwise there is a 
          # small window during which users who pip install scikit-image will require compilation. 
          twine upload ${{ github.workspace }}/dist/*.whl
```